cmd/sks_create.go

```diff
@@ -88,6 +84,22 @@ Supported output template annotations: %s`,
 			delete(addOns, "exoscale-cloud-controller")
 		}
 
+		version, err := cmd.Flags().GetString("kubernetes-version")
+		if err != nil {
+			return err
+		}
+		if version == "latest" {
+			ctx := exoapi.WithEndpoint(gContext, exoapi.NewReqEndpoint(gCurrentAccount.Environment, zone))
+			versions, err := cs.ListSKSClusterVersions(ctx)
+			if err != nil || len(versions) == 0 {
+				if len(versions) == 0 {
+					err = errors.New("no version returned by the API")
+				}
+				return fmt.Errorf("unable to retrieve SKS versions: %s", err)
+			}
+			version = versions[0]
```

Comment:
What do you think about extracting that code to a dedicated function? This way it should be simple to test it.

Reply:
I'm not sure it's worth testing this specific part, since it's mostly egoscale code which already tests it...